Languages/IronPython/IronPython/Lib/iptest/process_util.py

```diff
@@ -148,6 +151,8 @@ def kill_process(arg):
 
 def get_clr_dir():
     import clr
-    from System import Type
-    from System.IO import Path
-    return Path.GetDirectoryName(Type.GetType('System.Int32').Assembly.Location)
+    import System
+    if is_netstandard:
+        clr.AddReference("System.Runtime.Extensions")
+        return System.IO.Path.Combine(System.Environment.GetEnvironmentVariable("windir"), r"Microsoft.NET\Framework\v4.0.30319")
```

Comment:
This won't work on *NIX (I know you aren't trying to get it to work on *NIX right now, but just wanted to flag it).